### PR TITLE
Eq 16 submitter foundation

### PR DESCRIPTION
**Pull request by weapdiv-david**
**_What**_

A Simple app that receives messages from RabbitMQ Cluster and places them in a rabbitMQ bucket.

**_How to test**_
1) Log into the RabbitMQ Console for the environment you wish to test.
2) Verify the queue exists
3) Place a message on the queue using the RabbitMQ Admin
4) Verify that the message appears in your environments S3 Bucket

**_Who can test**_

Anybody except @weapdiv_david

### CODE REVIEW

**Comment by warrenbailey** [comment]
Looks good to me. I've seen it working and it does exactly what we want, so happy for it to be merged. 

Not 100% sure about that class copied from the pika site, though it does do what we need, so I'll let @dhilton make the final call.

**Comment by dhilton** [comment]
Can we squash some the commits please? 19c2084 and f9a7f5a seem to either need squashing or better commit messages as I can't work out what they change apart from looking at the code.

Also can the README explain how to install the application locally for dev and remotely for production usage?

**Comment by weapdiv-david** [comment]
The two commits in question...  The first simply removes some files that were part of the initial commit, but shouldn't have been, namely the .idea project files.  The second commit removes some unused code that was put in at the beginning of development but was no longer needed by the time it had all been refactored.  I'm struggling to rebase it though due to the addition and deletion of files.

File: remote_deploy.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+wget https://github.com/ONSdigital/eq-submitter/archive/master.zip
+unzip master.zip
+mv eq-submitter-master eq-submitter
+cd eq-submitter
+sudo pip install -r requirements.txt
+sudo service supervisor restart
```
Comment (dhilton) [inline remote_deploy.sh]: This doesn't copy the file as explained in the README - does this matter? Also this doesn't install the necessary packages by apt-get - can we add that in here so we can automate all of it?

**Pull request merged**